Header header
#add more for the lines and such, like in detections
ObjectsDetected[] objects
GoalpostsDetected[] goalposts
LinesDetected[] lines
ObstaclesDetected[] obstacles


